scripts/filestash/filestash_config.json.template

```diff
@@ -0,0 +1,90 @@
+{
+  "general": {
+      "name": null,
+      "port": null,
+      "host": null,
+      "secret_key": "zUMyrLIDRMJKEdWw",
```

Comment:
How is this generated, can we update this with our common secret? @GitHK

Reply:
It is just some random string. I can randomise within this script and replace .

Reply:
OK, I've added this